joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/LinkingUtil.scala

```diff
@@ -41,11 +36,9 @@ trait LinkingUtil {
   def nodesWithFullName(cpg: Cpg, x: String): mutable.Seq[NodeRef[_ <: NodeDb]] =
     cpg.graph.indexManager.lookup(PropertyNames.FULL_NAME, x).asScala
 
-  /** For all nodes `n` with a label in `srcLabels`, determine the value of `n.\$dstFullNameKey`, use that to lookup the
-    * destination node in `dstNodeMap`, and create an edge of type `edgeType` between `n` and the destination node.
-    */
-  def linkToSingle(
+  def newLinkToSingle(
```

Comment:
Could this not be overridden? `newLinkToSingle` is not a descriptive name. 
```suggestion
  def linkToSingle(
```

Reply:
I didn't change all the passes, so kept the original method as is. I am not sure whether it gets used inside private repos or not so chose to keep the original method as is. 

I will change other passes from where this method is being called.